CMakePCHCompiler.cmake

```diff
@@ -224,6 +204,12 @@ macro(__define_pch_compiler lang)
 			CMAKE_${lang}PCH_COMPILE_OBJECT
 			${CMAKE_${lang}PCH_COMPILE_OBJECT}
 			)
+		#2017: Enable MSVC2010 and forward to work.
+		#With /Zi Microsoft.Cpp.Win32.targets deletes the PCH file.
+		string(APPEND CMAKE_${lang}_FLAGS_DEBUG " /Z7")
```

Comment:
Do we really need such a drastic solution for MSVC? I understand this effectively turns off PDB generation and puts debug information in .obj files itself. This can be a really showstopper for many users, and has to be carefully documented in the README.

Reply:
Yeah, I would document it. As far do you need it? Well, yes, for MSVC2010 forward, which is all relevant version of MSVC. I'd be curious to know what the downside is, but if you can debug, and see quick build times, I don't see a problem. In any case, yes, it's not happening any other way.